### DISCUSSION TITLE
Unable to Resolve Action (Version 4.2.2)

### DISCUSSION THREAD

**Discussion by raeganbarker** [JamesIves/github-pages-deploy-action#1003] [Q&A] [answered]
Is anyone else running into issues using version `4.2.2`? I get the following error in my GitHub workflows when trying to use `4.2.2` but didn't have that issue with `4.1.4`.

```
Error: Unable to resolve action `JamesIves/github-pages-deploy-action@4.2.2`, unable to find version `4.2.2`
```

Thanks!

**Comment by JamesIves** [comment, marked as answer]
As of `4.2.2` the version tags will now include a `v` to be consistent with other officially provided actions by GitHub. You should be able to point your workflow to `JamesIves/github-pages-deploy-action@v4.2.2` and it will work. Additionally major version tags are now available by pointing to `@v4` (although not recommended to use).

I'm curious though, did you manually bump this version or did Dependabot create the PR that changed it from `4.2.1 -> 4.2.2`? In my tests Dependabot added the `v` automatically for me.

**Reply by raeganbarker** [reply]
Thanks for the quick, reply!

This was an update to a newly-configured repo (Dependabot wasn't set up, yet) so appears to have been user error on my part missing that prefix. I just checked the Dependabot PR for a different repo and it added the `v` as expected.

Thanks again!
